rhc-ose-ansible/inventory/ose-provision

```diff
@@ -9,9 +8,15 @@ openstack_key_name=""
 #openshift_storage_disk_volume=/dev/vdb
 #openstack_flavor_name="m1.medium"
 #openstack_image_name="ose3_1-base"
-#openshift_openstack_master_storage_size: 10
-#openshift_openstack_node_storage_size: 10
-
+#openshift_openstack_master_storage_size=10
+#openshift_openstack_node_storage_size=10
+#env_id=
+inv_cluster_method=native
```

Comment:
Sure agreed. I'm hesitant to go with `openshift_ansible...` as we don't want our vars to overload/conflict with variables they set or we may run into compatibility issues with some of their roles/playbooks